src/config/index.ts

```diff
@@ -112,6 +102,26 @@ export const BaseSettingsDefinition = {
     type: 'string',
     default: '',
   },
+  SUBSCRIPTION_RETRY_MAX_MS: {
```

Comment:
Is this the right name for this env var?
Asking because WS DPs usually require a _connection_ step and then a _subscribe_ (to assets) step. I'm worried this might mix up background execute subscriptions with the actual subscribe action after a WS connection.

Reply:
Updated to `STREAM_HANDLER_RETRY_` since we've moved it into the stream transport